kube/pod.go

```diff
@@ -68,8 +68,12 @@ func NewPodTemplate(role *model.Role, settings *ExportSettings) (helm.Node, erro
 	container.Add("readinessProbe", readinessProbe)
 	container.Sort()
 
+	imagePullSecrets := helm.NewMapping()
+	imagePullSecrets.Add("name", "registry-credentials")
+
 	spec := helm.NewMapping()
 	spec.Add("containers", helm.NewList(container))
+	spec.Add("imagePullSecrets", helm.NewList(imagePullSecrets))
```

Comment:
I have not actually tested this change yet, but I'm wondering if specifying "empty" secrets work, or if trying to log in with an empty username/password in a registry that doesn't support auth will throw an error.

Reply:
I considered putting a `Block` around this if the username is null in `values.yaml`, it might be necessary. I'll try setting up a local registry to test this.